prompb/rwcommon/codec_test.go

```diff
@@ -113,6 +113,7 @@ func testIntHistogram() histogram.Histogram {
 			{Offset: 0, Length: 1},
 		},
 		NegativeBuckets: []int64{1, 2, -2, 1, -1, 0},
+		CustomValues:    []float64{21421, 523}, // Technically not valid nhcb, but we test if fields would be copied.
```

Comment:
It's technically not a valid histogram, cause schema is 0 :)

But I think this is ok, since remote write is kind of dumb when it comes to histograms, it just needs to copy stuff and let tsdb validate later.

In fact you might want to use schema 1 to verify that schema is copied as well :)